models/events-model.js

```diff
@@ -191,6 +191,7 @@ if (patchBody.event_name) updateObj.event_name = patchBody.event_name;
 if (patchBody.event_date) updateObj.event_date = patchBody.event_date;
```

Comment:
can we make sure we are adding it as a date object
new Date(patchBody.event_date)